google-cloud/README.md

```diff
@@ -78,20 +99,32 @@ We'll use the Houston Python client to save the plan.
    export HOUSTON_KEY='<your api key>'
    ```
    **Never commit this key anywhere in your repo!**
+   
+   If you have a self-hosted solution then you will need another environment variable for the URL and port for your 
```

Comment:
Self hosted is the only option now so please change the wording here.

Also, please explain how they can find the base URL of their server, which is either by:
1. Going to Google Compute Engine and looking at the public IP, and setting it to `http://<public ip>/api/v1`
2. Going to secret manager in the cloud console and viewing the secret value
3. Using gcloud to print the secret (`gcloud secrets versions access latest --secret=houston-base-url`)

Reply:
Add the details mentioned for getting the base URL to the terraforming section if desired, it can be it's own seperate section if/when an alternative to the Terraform option is written.